src/components/App.js

```diff
@@ -0,0 +1,22 @@
+import ButtonPanel from './ButtonPanel';
+import Display from './Display';
+
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <h1>
+          Calculator
+        </h1>
+      </header>
+      <body>
+        <>
+          <Display />
+          <ButtonPanel />
+        </>
+      </body>
+    </div>
```

Comment:
- [ ] [REQUIRED] This code causes an JSX error. I think, `body` should not be a child of `div`:
![1](https://user-images.githubusercontent.com/58552748/127804214-6eb28828-0262-4e4e-984d-14f0dbe37834.gif)